Sources/Player/Mocks/Common/Types/StoredMediaProduct+Mock.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 
 public extension StoredMediaProduct {
-	static func mock(
+	static func storedMock(
```

Comment:
So, I remember back then that it was confusing the compiler and returning a `MediaProduct` instead of a `StoredMediaProduct`.

But tried again and it seems to be working, so maybe it was some weird Xcode cache issue.